tests/templates/kuttl/ldap-authentication/README.md

```diff
@@ -0,0 +1,5 @@
+# LDAP Authenticator Test
+
+This test sets an LDAP user called 'alice'.
+
+See `authcheck.py` for examples of authorized access.
```

Comment:
Does this have to be a readme or can it be a comment in the respective files?

Reply:
I love the idea of having more READMEs. However, in this case you're right. There is not that much to say about this test. We could remove that file alltogether and it would be okay.

Reply:
Fine with removing it!